src/signature_pad.js

```diff
@@ -92,11 +92,12 @@ SignaturePad.prototype.clear = function () {
   this._isEmpty = true;
 };
 
-SignaturePad.prototype.fromDataURL = function (dataUrl) {
+SignaturePad.prototype.fromDataURL = function (dataUrl, options) {
```

Comment:
After rewrite to ES6, you can probably set the default values for function parameters, so you could write it as 
```js
function (dataUrl, options = {}) {...}
```
and get rid of `opts` variable.